### DISCUSSION TITLE
Update to Italian locale [IT]

### DISCUSSION THREAD

**Discussion by zhaknafein** [aclap-dev/video-downloadhelper#955] [Language & Translation]
[it.txt](https://github.com/user-attachments/files/16056509/it.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
